### PR TITLE
Add information about multiplex samples to download section

### LINKED ISSUE #75: Document download structure for multiplex samples
This is beginning to firm up. For now, I want to make sure we track adding it to the documentation!

### PR DESCRIPTION
**Pull request by jashapiro**
Here I am adding a section about the multiplex downloads to the download 
documentation.  I tried to be fairly brief, but I think this reflects the plan we agreed upon. Let me know if there is more detail that you think is appropriate in this section, or additional links that might make sense.

I am leaving it as a draft for now, as I think we still need illustrations, as per https://github.com/AlexsLemonade/scpca-docs/issues/75#issuecomment-1162082576

This PR replaces #82, so that what I write can be reviewed by @allyhawkins.

Closes #75

### CODE REVIEW

**Comment by jashapiro** [comment]
I have also added/updated image files so they are all the same resolution. I wonder if they are a bit big on rendering, so I could add a width parameter (changing to html) if we think that is a good idea.

File: docs/download_files.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ These files are delivered as a zip file.
 
 For all downloads, sample folders (indicated by the `SCPCS` prefix) contain the files for all libraries (`SCPCL` prefix) derived from that biological sample.
 Most samples only have one library that has been sequenced.
+For multiplexed libraries, the sample folder name will be a comma-separated list of all samples found in the library files that the folder contains.
```
[the remark this answers is not in the document]
Comment (jashapiro) [reply docs/download_files.md:8]: Yep, I missed that one when I changed the lower reference!

File: docs/download_files.md [hunk 2]
```diff
@@ -80,6 +82,22 @@ See the {ref}`Experiment metadata <sce_file_contents:experiment metadata>` secti
 For projects with bulk RNA-seq data, the `bulk_metadata.tsv` file will be included for project downloads.
 This file will contain fields equivalent to those found in the `single_cell_metadata.tsv` related to processing the sample, but will not contain patient or disease specific metadata (e.g. `age`, `sex`, `diagnosis`, `subdiagnosis`, `tissue_location`, or `disease_timing`).
 
+## Multiplexed libraries
+
+For multiplexed libraries, where multiple biological samples were combined via cellhashing or similar technology, the organization of the downloaded files and metadata is slightly different.
```
[the remark this answers is not in the document]
Comment (jashapiro) [reply docs/download_files.md:87]: I realized that in most other places we talk about "multiplexed samples", but here I used "multiplexed libraries". Wondering if we should standardize these, and if so in which direction?

[the remark this answers is not in the document]
Comment (jashapiro) [reply docs/download_files.md]: I updated this to "multiplexed sample libraries" which I think makes things a bit more clear & consistent, & added a link to the FAQ

**Review by allyhawkins — APPROVED** [review]
LGTM!

**Pull request merged**